### ISSUE TITLE
Unable to Connect to proxy error

### ISSUE THREAD

**Issue by vaishakbhalli** [DigitalSecurity/btlejuice#22]
I get this error all along..."unable to connect to proxy". Do i need to configure it manually?Help and guidance on this matter would be deeply appreciated.
![cant connect to proxy](https://user-images.githubusercontent.com/29950251/27912965-80317528-6256-11e7-9f04-6628944abf85.png)

**Comment by virtualabs** [comment]
Did you start the bteljuice proxy manually ?

**Comment by vaishakbhalli** [comment]
i didnt configure any proxy externally.

**Comment by vaishakbhalli** [comment]
![screenshot from 2017-07-10 15-46-16](https://user-images.githubusercontent.com/29950251/28028436-691fb070-6594-11e7-83ce-8052d01b0647.png)
I tried this again today but didnt turn out to be successful.

**Comment by virtualabs** [comment]
You have to run *btlejuice-proxy* on a Rasp Pi equipped with a BT4 adapter
(for instance) and *btlejuice* with *-u* and *-w* on your kali, as you do.
As a matter of fact, if you run no *btlejuice-proxy* on your raspberry, the
tool won't connect because no port is open.

**Comment by vaishakbhalli** [comment]
yes i have done that. i have no error on the pi side but i get this error on the kali.

**Comment by vaishakbhalli** [comment]
Here's a quick rundown on the usage i have been following so far:
1.on Pi 3
sudo hciconfig
sudo service bluetooth start
sudo hciconfig hci0 up
sudo btlejuice-proxy

2.on Kali
sudo hciconfig 
sudo service bluetooth stop
sudo hciconfig hci1 up(using an external BT4 adapter since my laptop doesn't support BT4)
sudo btlejuice -u IP addr of PI -w -i 1

**Comment by bik3te** [comment]
Mmmm, it's strange. How are your rasp and the host connected ? Did you got a Bad perm error when installing btlejuice on the Rasp ?

Could you post a picture of the btlejuice-proxy command execution please ?

**Comment by vaishakbhalli** [comment]
i used the following command for the installation on both the Laptop and Pi 3: sudo npm install --unsafe-perm --verbose -g btlejuice. Will post the btlejuice-proxy part picture in a while.

**Comment by vaishakbhalli** [comment]
I removed the btlejuice  using this cmd "sudo npm remove -g btlejuice" and when i try to install it from the scratch, i now encounter the following error:
![screenshot from 2017-07-11 13-20-18](https://user-images.githubusercontent.com/29950251/28068137-1667c7d4-663c-11e7-850d-f7ef4891f8ce.png)

**Comment by vaishakbhalli** [comment]
I removed have removed the Btlejuice on the Laptop but while installing the dependencies , I got the above error.should I proceed with the installation?

**Comment by bik3te** [comment]
You can try, it won't get worse but your repos are broken...

**Comment by vaishakbhalli** [comment]
So hw do I fix them?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by bik3te** [comment]
vaishakbhalli, I'm sorry but I don't understand how you put yourself in this situation. Anyway, this doesn't seem related to btlejuice. We can't help you for that, better look for solution by googling the error and try to repair it. Or reproducing the install of btlejuice on a fresh ubuntu install. Good luck
